layouts/partials/carousel/host.css

```diff
@@ -22,9 +22,10 @@
 ".tvp-carousel-holder > iframe {" +
     "top: 0;" +
     "left: 0;" +
-    "width: 100%;" +
+    "width: 1px;" +
+    "min-width: 100%;" +
     "height: 100%;" +
-    "position: absolute;" +
+    "position: relative;" +
```

Comment:
Changing the carousel to be relative? please share what's causing this change.
Also, if we really need this to be relative then `top:0; left: 0;` is not necessary.

Reply:
yes, this was for a fix for IOS devices, carousel was chopped on the right side, im trying to find the ticket.
will remove unused code.

Reply:
I get it now, yes remove unused code, no need to find ticket thanks.